Sources/VLCActionSheet/VLCActionSheetCell.swift

```diff
@@ -19,12 +19,23 @@ class VLCActionSheetCell: UICollectionViewCell {
         icon.contentMode = .scaleAspectFit
         return icon
     }()
+    
+    let checkmark: UILabel = {
+        let checkmark = UILabel()
+        checkmark.text = "✓"
+        checkmark.font = UIFont.systemFont(ofSize: 15)
```

Comment:
Should this be bigger?

Reply:
Indeed, It's a bit small but I wonder if this shouldn't be a separate class. For example perhaps a `VLCSettingSheetCell`?

Reply:
Also, open thought, I'm aware that on Figma the checkmark is on the right but shouldn't we consider putting the check mark on the left like the edit mode? Might just be a personal preference but I think it will be nicer 😅

Reply:
Not all ActionSheet cells need/will need checkmarks.

Reply:
So for the few that need both checkmarks and images, it should be placed on the right, no?

Reply:
Just subclassed it to `VLCSettingSheetCell` and it makes more sense :D